src/checkers/english.rs

```diff
@@ -2,8 +2,12 @@
 use crate::storage;
 
 // given an input, check every item in the array and return true if any of them match
-pub fn check_english(input: &str) -> bool {
-    storage::STORAGE.contains(input)
+pub fn check_english(input: &str) -> Option<&str> {
+    if let Some(result) = storage::DICTIONARIES.iter().find(|(_, words)| words.contains(input)) {
+        // result.0 is filename
+        return Some(result.0)
```

Comment:
We want to return the word + filename here. This is why I wanted to return a success object as it can let us add in more info :D